app/main/views/search.py

```diff
@@ -204,6 +195,21 @@ def supplier_search():
 
     pages = get_page_list(SUPPLIER_RESULTS_PER_PAGE, num_results, page)
 
+    SELLER_TYPE_KEYS = [
+        'start_up',
+        'sme',
+        'nfp_social_enterprise',
+        'product',
+        'indigenous'
+    ]
+
+    seller_type_filters = {st: st in selected_seller_types for st in SELLER_TYPE_KEYS}
```

Comment:
at this stage we don't.. however ideally we'd populate the seller type options from the server side as we do with the domain list. 

will take it out now and can enhance in the longer term.